src/quacc/recipes/common/phonons.py

```diff
@@ -5,34 +5,36 @@
 from importlib.util import find_spec
 from typing import TYPE_CHECKING
 
+from ase.atoms import Atoms
 from monty.dev import requires
 
 from quacc import job, subflow
-from quacc.atoms.phonons import get_phonopy, phonopy_atoms_to_ase_atoms
+from quacc.atoms.phonons import (
+    get_atoms_supercell_by_phonopy,
+    get_phonopy,
+    phonopy_atoms_to_ase_atoms,
+)
 from quacc.runners.phonons import PhonopyRunner
 from quacc.schemas.phonons import summarize_phonopy
+from quacc.utils.dicts import recursive_dict_merge
 
 has_phonopy = bool(find_spec("phonopy"))
 has_seekpath = bool(find_spec("seekpath"))
 
 if TYPE_CHECKING:
     from typing import Any
 
-    from ase.atoms import Atoms
-
     from quacc import Job
     from quacc.types import PhononSchema
 
-    if has_phonopy:
-        from phonopy import Phonopy
-
 
 @subflow
 @requires(has_phonopy, "Phonopy must be installed. Run `pip install quacc[phonons]`")
 @requires(has_seekpath, "Seekpath must be installed. Run `pip install quacc[phonons]`")
 def phonon_subflow(
-    atoms: Atoms,
+    displaced_atoms: Atoms,
     force_job: Job,
+    non_displaced_atoms: Atoms | None = None,
```

Comment:
Out of curiosity, what is the motivation to have two separate `Atoms` objects rather than one `Atoms` objects for the full thing and specific indices to not displace? It seems like it might be a bit awkward if they are separate `Atoms` objects because wouldn't the user have to split up their `Atoms` object into two before calling this function? I'm sure there's a reason for your suggestion --- just trying to pick up on what it is. We may have gone over this a long time ago.

Reply:
In commit "https://github.com/Quantum-Accelerators/quacc/pull/2110/commits/24608c9046a1b0a8cb2bbfaf0691e5a2fc60e127" I left a comment.

I think we moved out of this approach to the "FixAtoms" approach, but that did not work as well so ended up here.

Reply:
@tomdemeyere: Apologies if we're revisiting history here (it's been a while...). I'm not sure the commit shows the commit, but my question is mostly about what if we had an `Atoms` object as input with a `list[int]` of indices to not displace. Was there an issue with that approach? If so, we can stick with what you have here